src/components/PostInfo/PostInfo.tsx

```diff
@@ -0,0 +1,21 @@
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+import './PostInfo.scss';
+
+type Props = {
+  post: Post;
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => (
+  <>
+    <h2 className="post__title">{post.title}</h2>
+    <p className="post__body">{post.body}</p>
+    {post.user && (
+      <UserInfo user={post.user} />
+    )}
+    {post.comments.length > 0 && (
```

Comment:
```suggestion
    {post.comments.length && (
```

Reply:
I tried not to write > 0 and for some reason it renders 0